packages/grid/vpn/tailscale.sh

```diff
@@ -1,8 +1,15 @@
 #!/bin/sh
 
+# iptables --list
 # block all traffic but port 80 on the tailscale0 interface
 # iptables -A INPUT -i tailscale0 -p tcp --dport 80 -j ACCEPT
 # iptables -A INPUT -i tailscale0 -p tcp -j REJECT
+# iptables -A OUTPUT -p tcp --destination-port 4000 -j DROP
+
+iptables -A INPUT -i tailscale0 -p tcp --destination-port 4000 -j REJECT
+iptables -A INPUT -i tailscale0 -p tcp --destination-port 8001 -j REJECT
+iptables -A INPUT -i tailscale0 -p tcp --destination-port 8011 -j REJECT
+iptables -A INPUT -i tailscale0 -p tcp --destination-port 8080 -j REJECT
```

Comment:
these ports are from known services we bind to the same network interface to allow intercommunication between containers while providing a shared outgoing interface over the vpn.